algebra-jackson/src/test/java/com/hubspot/algebra/ResultModuleTest.java

```diff
@@ -79,6 +79,22 @@
   private static final Result<String, NullValue> NULL_ERR = Result.nullErr();
   private static final String NULL_ERR_JSON = "{\"@error\":null,\"@result\":\"ERR\"}";
 
+  private static final Result<Map<String, Result<TestBean, TestError>>, Map<String, Result<TestBean, TestError>>> NESTED_OK_OK =
```

Comment:
Can you please simplify these types? Combining the maps and results this way really makes it hard to grok this stuff.